### ISSUE TITLE
Password field is not visible when logging out of second account

### ISSUE THREAD

**Issue by fossfreedom** [linuxmint/slick-greeter#99]
```
 * slick-greeter version 1.1.4 + mint 19 unstable
 * Distribution - ubuntu budgie 18.04
 * virtualbox
 * both 32 & 64 bit
 ```

**Issue**
On logout of a second user account the password field is not visible

**Steps to reproduce**
Create a second account.  Login.  Logout

**Expected behaviour**
The password field should be visible - if logging out of a machine with just one account, the password field is visible.

With the second account you have to click back to a previous account and then back again to show the password field.

**Other information**

see screenshot

![budgie test 64bit running - oracle vm virtualbox_048](https://user-images.githubusercontent.com/996240/36944363-6ae49354-1f92-11e8-89ee-c3461e385e9e.png)

**Comment by jbjet777** [comment]
I'm also experiencing this issue on Linux Mint 19, has there been any movement on this bug?

**Comment by yukithecat** [comment]
I have the exact same issue, up to date Arch Linux with Xfce. Reproduced same way as in fossfreedom's post above.

**Comment by darmeth** [comment]
This issue is still present in slick-greeter 1.2.4. It is intermittent and does not accur on every single startup. Also, this does not happen on machines with only one user account. I have tested on Fedora, Mint and Manjaro all running 1.2.4 and the issue prevails.

**Comment by mkauzlar** [comment]
I experienced this issue from Mint 18 onward.
I have now installed Mint 19.2 with (which has slick-greeter/tina,now 1.2.7+tina amd64 [installed])
and the problem is still there.

**Comment by hpg-mheck** [comment]
Confirmed 15NOV2019 on CentOS 7 on ARM.

**Comment by LinuxOnTheDesktop** [comment]
I created a new user and I find that I can log into without even being prompted for the password. I set the password via Mint's GUI. Then I set the password with `passwd`; it made no difference. This is a security issue.

**Comment by tomocafe** [comment]
I’ve also been having this issue on Solus for a long time now.

**Comment by icarter09** [comment]
@fossfreedom  are you still experiencing this issue?

**Comment by jbjet777** [comment]
Yep, all three machines.  All latest updates and latest kernel (5.4) on LM19.3

**Comment by icarter09** [comment]
@jbjet777 thanks for the update.

**Comment by darmeth** [comment]
Yes, although this is still intermittent. For me, this happens only on machines with 2 or more user accounts. Single accounts are unaffected. Also, I've only seen this at start up or after reboot; never on suspend.

**Comment by jbjet777** [comment]
Same here.  I did not have the issue when only one account was on the machines, however, I do have the issue in **all** circumstances, reboot, resume, and suspend.  Second user listed on Slick Greeter only, not the first user.  Using Intel video graphics on all machines, if that makes a difference.

I can get the input box to return, by selecting the "desktop environment switcher" icon and immediately deselecting it (not actually switch the desktop environment).

I'm positive this is a refresh issue upon user selection.  Something is not being redrawn or refreshed upon selecting the "second" user.

**Comment by LinuxOnTheDesktop** [comment]
Here is a possibly-related problem. I created a new user and assigned a password. I used the GUI to do this but admittedly I think that later I used `passwd` to change the password. The problem is: Mint is allowing me to log into the account without entering any password. Indeed it does not even prompt me for a password. I found the using the GUI to change the password - to, er, what my present password for the account was already - fixed the problem. Should I open a new issue for this problem, please?

**Comment by LinuxOnTheDesktop** [comment]
@icarter09 Has enough information been gathered now, please?
